File: teste/solo/app/src/main/java/com/example/solo/network/ApiService.java
package com.example.solo.network;

import com.example.solo.models.SoloUser;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface ApiService {

    @POST("/login")
    Call<SoloUser> login(@Body Map<String, String> body);
}

File: solo/app/src/main/java/com/example/solo/model/api/ApiService.kt
package com.example.solo.model.api

import com.example.solo.model.data.LoginRequest
import com.example.solo.model.data.LoginResponse
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface ApiService {
    @POST("login")
    fun login(@Body loginRequest: LoginRequest): Call<LoginResponse>
}
